packages/themes/src/themes.js

```diff
@@ -0,0 +1,66 @@
+import * as g10 from './g10';
+import * as g100 from './g100';
+import * as white from './white';
+import * as g90 from './g90';
+import * as v9 from './v9';
+
+function rgbToHex(rgbString) {
+  return (
+    '#' +
+    rgbString
+      .match(/\d+/g)
+      .map((x) => parseInt(x).toString(16).padStart(2, '0'))
+      .join('')
+  );
+}
+
+export { g10, g90, g100, white, v9 };
+/**
+ * Dynamically determines and returns the current theme token map
+ *
+ * @param {node} domNode the dom node to examine to determine the current theme
+ * @param {string} cssVariablePrefix the prefix of the carbon css variables that was set at mixin time
+ * @returns {object} object with current theme tokens, null if we can't determine the current theme
+ */
+export function determineCurrentTheme(domNode, cssVariablePrefix) {
```

Comment:
```suggestion
export function getCurrentTheme(domNode, cssVariablePrefix) {
```
?

Reply:
seems like it's actually returning the full theme

Reply:
I fixed this @theiliad